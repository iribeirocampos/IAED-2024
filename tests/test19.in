p teste1 20 1 5 6
e teste1 11-JJ-11 01-01-2024 10:00
e teste1 11-JJ-ll 01-01-2024 11:00
e teste1 11-jj-11 01-01-2024 11:00
e teste1 11-11-11 01-01-2024 11:00
e teste1 JJ-JJ-JJ 01-01-2024 11:00
e teste1 11-JJ-!= 01-01-2024 11:00
e teste 11-JJ-!= 01-01-2022 09:00
e teste1 11-JJ-!= 01-01-2022 09:00
e teste1 11-JJ-11 01-01-2022 09:00
e teste1 11-JJ-JJ 01-01-2022 09:00
e teste1 11-JJ-JJ 01-01-2024 09:00
e teste1 AA-JJ-11 02-02-2024 10:00
e teste1 AA-JJ-12 02-02-2024 10:00
e teste1 AA-JJ-13 02-02-2024 10:00
e teste1 AA-JJ-14 02-02-2024 10:00
s teste1 AA-JJ-11 02-02-2024 10:00
s teste1 AA-JJ-12 02-02-2024 10:00
s teste1 AA-JJ-13 02-02-2024 10:00
s teste1 AA-JJ-14 02-02-2024 10:00
e teste1 AA-JJ-1A 03-02-2024 10:00
q