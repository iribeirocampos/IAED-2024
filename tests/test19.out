teste1 19
11-JJ-ll: invalid licence plate.
11-jj-11: invalid licence plate.
11-11-11: invalid licence plate.
JJ-JJ-JJ: invalid licence plate.
11-JJ-!=: invalid licence plate.
teste: no such parking.
11-JJ-!=: invalid licence plate.
11-JJ-11: invalid vehicle entry.
invalid date.
invalid date.
teste1 18
teste1 17
teste1 16
teste1 15
AA-JJ-11 02-02-2024 10:00 02-02-2024 10:00 0.00
AA-JJ-12 02-02-2024 10:00 02-02-2024 10:00 0.00
AA-JJ-13 02-02-2024 10:00 02-02-2024 10:00 0.00
AA-JJ-14 02-02-2024 10:00 02-02-2024 10:00 0.00
AA-JJ-1A: invalid licence plate.
